terraform/README.md

```diff
@@ -0,0 +1,23 @@
+
+```terraform
+provider "kubernetes" {
+
+}
+
+module "pomerium_ingress_controller" {
+    source = "git:https://github.com/pomerium/ingress-controller//terraform?ref=v0.28.0"
+}
+
+resource "kubernetes_manifest" "pomerium_config" {
```

Comment:
the installation and configuration need be split into separate steps, due to https://github.com/hashicorp/terraform-provider-kubernetes/issues/1367

Reply:
Can also use a `kubectl_manifest` instead of a kubernetes_manifest, which avoids this issue.

Reply:
Can also use `kubectl_manifest` instead of `kubernetes_manifest`, which doesn't have this issue.

`resource "kubectl_manifest" "pomerium_config" {
  yaml_body = <<YAML
apiVersion: ingress.pomerium.io/v1
kind: Pomerium
metadata:
  name: global
spec:
  secrets: pomerium-ingress-controller/bootstrap
YAML
}`